robottelo/datafactory.py

```diff
@@ -133,6 +150,31 @@ def valid_data_list():
 
 
 @datacheck
+def valid_emails_list():
+    """Returns a list of valid emails."""
+    return [
+        u'{0}@example.com'.format(gen_string('alpha')),
+        u'{0}@example.com'.format(gen_string('alphanumeric')),
+        u'{0}@example.com'.format(gen_string('numeric')),
+        u'{0}@example.com'.format(gen_string('alphanumeric', 48)),
+        u'{0}+{1}@example.com'.format(gen_alphanumeric(), gen_alphanumeric()),
+        u'{0}.{1}@example.com'.format(gen_alphanumeric(), gen_alphanumeric()),
```

Comment:
Nothing but just ease of use of using gen_alphanumeric.  But I agree, it should be consistently used.